apps/teams-test-app/src/components/privateApis/VideoExAPIs.tsx

```diff
@@ -80,22 +86,54 @@ const CheckIsSupported = (): React.ReactElement =>
     },
   });
 
-const RegisterForVideoFrame = (): React.ReactElement =>
+const MediaStreamRegisterForVideoFrame = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'videoExMediaStreamRegisterForVideoFrame',
+    title: 'medisStream - registerForVideoFrame',
+    onClick: async (setResult) => {
+      try {
+        const audioInferenceModel = new ArrayBuffer(8);
+        const view = new Uint8Array(audioInferenceModel);
+        for (let i = 0; i < view.length; i++) {
+          view[i] = i;
+        }
+        videoEx.registerForVideoFrame({
+          videoFrameHandler: async (frame) => {
+            setResult('video frame received');
+            return frame.videoFrame;
+          },
+          videoBufferHandler: (buffer) => buffer,
+          config: {
+            format: video.VideoFrameFormat.NV12,
+            requireCameraStream: false,
+            audioInferenceModel,
+          },
+        });
+      } catch (error) {
+        return `Faild to register for video frame: ${JSON.stringify(error)}`;
```

Comment:
```suggestion
        return `Failed to register for video frame: ${JSON.stringify(error)}`;
```

Reply:
hmm, fixing this typo will cause e2e failure, because hub-sdk also checks the typo string. Maybe I should create a new input with the typo fix and update the hub sdk and then remove this typo one

Reply:
fixed